SPECS/sudo/sudo.spec

```diff
@@ -1,18 +1,23 @@
 Summary:        Sudo
 Name:           sudo
-Version:        1.9.5p2
-Release:        3%{?dist}
+Version:        1.9.10
```

Comment:
I think we missed updating `cgmanifest.json`

Reply:
Yep.  Fixed,